### DISCUSSION TITLE
Pop-up menu with WEIRD suggestions in the chat input field

### DISCUSSION THREAD

**Discussion by maticpeprnyx** [community/community#58761] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hello everyone,

I am using Visual Studio Code Insiders with the Copilot Chat extension.

When I type in the chat input field, I get these strange suggestions that only match some of the letters I'm typing. They appear in a pop-up menu and kinda look like Intellisense suggestions but they're weird. It's hyphenated phrases, and the first word is always something like "mi" or "mat." The list changes dynamically with each typed character.

They certainly aren't natural language and they're super unhelpful when I try to ask natural language questions. The most annoying part is when the suggestion gets accepted upon hitting enter (you know, the key that also submits the question to the chatbot?). The pop-up menu only disappears after hitting space.

I have no idea if this is the result of extension that I have installed without realising it or what is going on.

Anybody else got the same problem?

**Comment by abulka** [comment]
I get this too - I'm suprised more people aren't complaining about it.  

For me, it seems to be matching user snippets names from all possible user snippets because I see (User Snippet) often in the annoying popup.

I have raised an issue in https://github.com/microsoft/vscode-copilot-release/issues/331

**Comment by maticpeprnyx** [comment]
I managed to disable it by disabling the Angular Language Service extension in my editor.

**Comment by abulka** [comment]
I don't have the Angular Language Service extension installed. I still get the annoying copilot chat window popups.
